canonical-websites/www.ubuntu.com
# 4068 update livepatch

## Done

- Moved /server/livepatch to /support/livepatch
- Added a redirect
- Also redirected /livepath
- Update the content per the [copy doc](https://docs.google.com/document/d/1Ofw9bQ4I40gy9hDIjuDMApz6sDK8qSdHI03-oY7xs-A/edit#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/support/livepatch](http://0.0.0.0:8001/support/livepatch)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the copy doc and the design
- test the redirects for [http://0.0.0.0:8001/server/livepatch](http://0.0.0.0:8001/server/livepatch) and [http://0.0.0.0:8001/livepatch](http://0.0.0.0:8001/livepatch)

## Issue / Card

Fixes #4068
